common/errors.js

```diff
@@ -142,12 +142,16 @@
                 window.location.reload();
             }; 
             if (exceptionFlag || window.location.pathname.indexOf('/search/') != -1 || window.location.pathname.indexOf('/viewer/') != -1) return;
-
+            
             if (ERMrest && exception instanceof ERMrest.UnauthorizedError || exception.code == errorNames.unauthorized) {
```

Comment:
While you are making changes here in the `handleException` function, can you change this line like the one you added with better parenthetic grouping?

Reply:
Should read something like:
```
( (ERMrest && exception instanceof ERMrest.UnauthorizedError) || exception.code == errorNames.unauthorized )
```